liquibase-core/src/main/java/liquibase/parser/ChangeLogParserConfiguration.java

```diff
@@ -12,7 +12,7 @@ public class ChangeLogParserConfiguration implements AutoloadedConfigurations {
     public static final ConfigurationDefinition<Boolean> USE_PROCEDURE_SCHEMA;
     public static final ConfigurationDefinition<MissingPropertyMode> MISSING_PROPERTY_MODE;
     public static final ConfigurationDefinition<ChangelogParseMode> CHANGELOG_PARSE_MODE;
-    public static final ConfigurationDefinition<Boolean> WARN_OR_ERROR_ON_MISSING_CHANGELOGS;
+    public static final ConfigurationDefinition<Boolean> WARN_OR_ERROR_ON_MISSING_INCLUDE_FILE;
```

Comment:
The name now is much more expressive, thanks.

Reply:
Hey @mhewedy, I just want to let you know as you can see on the latest changes I have updated this configuration one more time based on the team's code review. I have renamed it to `ON_MISSING_INCLUDE_FILE`, and also changed it to be set with an Enumeration which for now only has WARN and FAIL values. I think it will now be even more self-explainable. I hope you are happy with these changes.  

Thanks, 
Daniel.